### PR TITLE
bpo-41933: Clarify wording for s * n in Common Sequence Operations

### PR DESCRIPTION
**Pull request by Sipty**
https://bugs.python.org/issue41933

### CODE REVIEW

File: Misc/NEWS.d/next/Documentation/2020-10-05-20-04-43.bpo-41933.Pff94-.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Clarified wording of s * n in the Common Sequence Operations
```
Comment (eamanu) [inline Misc/NEWS.d/next/Documentation/2020-10-05-20-04-43.bpo-41933.Pff94-.rst:1]: I think NEWS is not neceesary here. But we need wait for a core dev opinion

**Review by eamanu — APPROVED** [review]
LGTM

**Review by JulienPalard — APPROVED** [review]
I prefer it this way too :+1:

**Comment by JulienPalard** [comment]
Thanks @Sipty !

**Pull request merged**